identity-service/sequelize/migrations/20211223183608-add-solana-notification-indexes.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (transaction) => {
+      await queryInterface.addIndex('SolanaNotifications', ['userId'], { transaction })
+      await queryInterface.addIndex('SolanaNotificationActions', ['slot'], { transaction })
+      await queryInterface.addIndex('SolanaNotificationActions', ['notificationId'], { transaction })
+      await queryInterface.addIndex('SolanaNotificationActions', ['notificationId', 'actionEntityType', 'actionEntityId', 'slot'], { transaction })
+    })
+  },
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.removeIndex('SolanaNotifications', ['userId'])
```

Comment:
can we do the down as a transaction as well? so it's either all or nothing?

Reply:
fair point 👍